hcipy/aperture/realistic.py

```diff
@@ -22,11 +26,17 @@ def make_vlt_aperture(normalized=False, telescope='ut3', with_spiders=True, with
 		If this is True, a cover will be created for the M3 in stowed position.
 		This M3 cover is only available on UT4, mimicking the ERIS pupil. A warning
 		will be emitted when using an M3 cover with other UTs. Default: False.
+	with_segment_gaps : boolean
```

Comment:
This argument was removed from the signature. Remove it from the docstring as well.